python/lsst/obs/decam/instrument.py

```diff
@@ -84,5 +86,15 @@ def getRawFormatter(self, dataId):
         return DarkEnergyCameraRawFormatter
 
     def writeCuratedCalibrations(self, butler):
-        # TODO: DM-21016
-        pass
+        """Write human-curated calibration Datasets to the given Butler with
+        the appropriate validity ranges.
+
+        This is a temporary API that should go away once obs_ packages have
+        a standardized approach to this problem.
+        """
```

Comment:
I think we should remove this docstring and leave documenting this method to the parent class.